lib/views/search_view.dart

```diff
@@ -13,30 +11,13 @@ class _SearchState extends State<Search> {
   Widget build(BuildContext context) {
     final double height = MediaQuery.of(context).size.height;
     final double width = MediaQuery.of(context).size.width;
-    return SafeArea(
-      child: Padding(
-        padding: const EdgeInsets.symmetric(vertical: 12),
-        child: Column(
-          children: <Widget>[
-            RelicBazaarStackedView(
-              height: height * 0.06,
-              width: width * 0.9,
-              upperColor: Colors.white,
-              child: TextFormField(
-                style: const TextStyle(
-                  fontFamily: 'pix M 8pt',
-                  fontSize: 18,
-                  color: RelicColors.primaryBlack,
-                ),
-                decoration: const InputDecoration(
-                    hintText: 'search for categories, items and more...',
-                    border: InputBorder.none,
-                    prefixIcon: Icon(RelicIcons.search)),
-              ),
-            ),
-          ],
-        ),
+    return Padding(
+      padding: EdgeInsets.only(
```

Comment:
Add uniform padding here, or put the search bar in AppBar here instead of padding.